test/simplifyExpression/basicsSearch/simplifyDoubleUnaryMinus.test.js

```diff
@@ -5,9 +5,9 @@ const testSimplify = require('./testSimplify');
 
 
 describe('simplifyDoubleUnaryMinus', function() {
-	var tests = [
-    	['--5', '5'],
-    	['--x', 'x']
+  var tests = [
+      ['--5', '5'],
```

Comment:
can you change this to just 2 spaces? same for `testSimplify.js`

I'm adding this to be put in the linter eventually ( #44 ) so it's easier to catch in the future